content/src/content/jcr_root/apps/core/wcm/components/navigation/v1/navigation/_cq_design_dialog/.content.xml

```diff
@@ -47,15 +47,16 @@
                                 forceSelection="{Boolean}true"
                                 name="./navigationRoot"
                                 required="{Boolean}true"/>
-                            <skipNavigationRoot
+                            <rootLevel
                                 jcr:primaryType="nt:unstructured"
-                                sling:resourceType="granite/ui/components/coral/foundation/form/checkbox"
-                                fieldDescription="Exclude the navigation root in the resulting tree, include its descendants only."
-                                text="Exclude navigation root"
-                                name="./skipNavigationRoot"
-                                value="{Boolean}true"
-                                checked="{Boolean}true"
-                                uncheckedValue="{Boolean}false"/>
+                                sling:resourceType="granite/ui/components/coral/foundation/form/numberfield"
+                                fieldDescription="The levels below navigation root that are to be excluded. To include navigation root enter 0."
+                                fieldLabel="Exclude Root Levels"
+                                max="100"
+                                min="0"
+                                name="./rootLevel"
+                                step="1"
+                                value="1"/>
```

Comment:
Can we set the value here based on `skipNavigationRoot` (if that was defined previously)?

Reply:
I wasn't referring to that, just that if you previously had selected `skipNavigationRoot==false` then you should see `rootLevel==0` in the dialog.

Reply:
Got it.